### PR TITLE
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute

### PR DESCRIPTION
**Pull request by alexbuckgit**
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute

**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by JustPies** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
@333fred @CyrusNajmabadi @gewarren @jcouv @KathleenDollard @mairaw @nschonni @reduckted @sharwell @terrajobst @v-thpra @Youssef1313

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

File: Language-Version-History.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Features Added in VB Language Versions
 
-*NOTE:* See how to [specify the language version for your project here](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/configure-language-version)
+*NOTE:* See how to [specify the language version for your project here](/dotnet/visual-basic/language-reference/configure-language-version)
```
Comment (gewarren) [inline Language-Version-History.md:3]: @BillWagner Do you take these sorts of docs-platform specific changes in the C# spec repo?

**Comment by 333fred** [comment]
See https://github.com/dotnet/csharpstandard/pull/613#issuecomment-1217021721. I believe this repo is the same, and this PR should be closed.

File: Language-Version-History.md (hunk 1 above)
Comment (BillWagner) [inline Language-Version-History.md:3]: We should keep the absolute link, so developers can navigate from the GitHub source easily. However, we should remove the locale:

```suggestion
*NOTE:* See how to [specify the language version for your project here](https://docs.microsoft.com/dotnet/visual-basic/language-reference/configure-language-version)
```

**Review by BillWagner — APPROVED** [review]
Thanks @alexbuckgit 

As with the other dependent repos, we want to keep the absolute links to docs.microsoft.com, but we do want to remove the locales.

I made a suggestion on this one, and once I commit it, I'll :shipit:

**Pull request merged**